content/concepts/paranoia_levels.md

```diff
@@ -43,7 +43,7 @@ It's important to think about a service's security requirements. The difference
 Running at  the highest paranoia level, PL 4, may seem appealing from a security standpoint, but *it could take many weeks to tune away the false positives encountered*. It is crucial to have enough time to fully deal with all false positives.
 
 {{% notice warning %}}
-Failure to properly tune an installation runs the risk of exposing users to a vast number of false positives. This can lead to a poor user experience, and might ultimately lead to a decision to completely disable the Core Rule Set. As such, **setting a high PL in blocking mode *without* adequate tuning to deal with false positives is very risky**.
+Failure to properly tune an installation runs the risk of exposing users to a vast number of false positives. This can lead to a poor user experience, and might ultimately lead to a decision to completely disable the CRS. As such, **setting a high PL in blocking mode *without* adequate tuning to deal with false positives is very risky**.
```

Comment:
```suggestion
Failure to properly tune an installation runs the risk of exposing users to a vast number of false positives. This can lead to a poor user experience, and might ultimately lead to a decision to completely disable CRS. As such, **setting a high PL in blocking mode *without* adequate tuning to deal with false positives is very risky**.
```